<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClangTools::Internal::SettingsWidget</class>
 <widget class="QWidget" name="ClangTools::Internal::SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Executables</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Clang-Tidy:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="Utils::PathChooser" name="clangTidyPathChooser" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="clazyStandaloneLabel">
        <property name="text">
         <string>Clazy-Standalone:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="Utils::PathChooser" name="clazyStandalonePathChooser" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ClangTools::Internal::RunSettingsWidget" name="runSettingsWidget" native="true"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>183</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ClangTools::Internal::RunSettingsWidget</class>
   <extends>QWidget</extends>
   <header>clangtools/runsettingswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
